server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/resolvers/TargetsLanguageOptionsResolver.java

```diff
@@ -97,6 +98,8 @@ private BazelProcess queryBazel(List<String> targets) {
     List<String> inputs = actionGraphParser.getInputsAsUri(target, bazelData.getExecRoot());
 
     return actionGraphParser.getOutputs(target, ACTION_GRAPH_SUFFIXES).stream()
+        .max(Comparator.naturalOrder())
+        .stream()
```

Comment:
to reflect that we are returning only one element now it would be nice to leave Optional here, and add `.stream()` it the final `flatMap` in the `getResultItemsForTargets` method

Reply:
right, fixed